[2021-10-08 00:38:47,570] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Accurate 2021-10-07T00:00:00+00:00 [queued]>
[2021-10-08 00:38:47,638] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Accurate 2021-10-07T00:00:00+00:00 [queued]>
[2021-10-08 00:38:47,640] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-08 00:38:47,641] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-08 00:38:47,643] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-08 00:38:47,666] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): Accurate> on 2021-10-07T00:00:00+00:00
[2021-10-08 00:38:47,692] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Accurate', '2021-10-07T00:00:00+00:00', '--job-id', '162', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp6a2qzqaf', '--error-file', '/tmp/tmpnewk6vjh']
[2021-10-08 00:38:47,695] {standard_task_runner.py:77} INFO - Job 162: Subtask Accurate
[2021-10-08 00:38:47,683] {standard_task_runner.py:52} INFO - Started process 5244 to run task
[2021-10-08 01:02:24,339] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Accurate 2021-10-07T00:00:00+00:00 [running]> on host d6b4131456ad
[2021-10-08 01:02:24,597] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Accurate
AIRFLOW_CTX_EXECUTION_DATE=2021-10-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-07T00:00:00+00:00
[2021-10-08 01:02:24,601] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-08 01:02:24,610] {subprocess.py:63} INFO - Running command: ['bash', '-c', "printf '============\n\nAccurate\n\n=============' "]
[2021-10-08 01:02:24,689] {subprocess.py:74} INFO - Output:
[2021-10-08 01:02:24,738] {subprocess.py:78} INFO - ============
[2021-10-08 01:02:24,740] {subprocess.py:78} INFO - 
[2021-10-08 01:02:24,741] {subprocess.py:78} INFO - Accurate
[2021-10-08 01:02:24,742] {subprocess.py:78} INFO - 
[2021-10-08 01:02:24,744] {subprocess.py:78} INFO - =============
[2021-10-08 01:02:24,746] {subprocess.py:82} INFO - Command exited with return code 0
[2021-10-08 01:02:24,846] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Accurate, execution_date=20211007T000000, start_date=20211008T003847, end_date=20211008T010224
[2021-10-08 01:02:24,940] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-08 01:02:25,007] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
